README.md

```diff
@@ -107,6 +107,28 @@ experiment with the changes live.)
 ```sass
     @import "filename";
 ```
+<hr />
+
+### Updating npm modules
```

Comment:
I don't think updating all the npm modules is necessary unless there is a big change in those modules. Most of the dependencies used in the boilerplate are `devDependencies` and won't affect anything in the production. Can you please remove this section. The other things looks good to me.

Reply:
https://github.com/shaminmeerankutty/webpack-sass-bootstrap-boilerplate/pull/12/commits/2700fe89e8ce84fdb0b045c360bcd91dd07800cb